src/components/Help/Documentation/index.tsx

```diff
@@ -0,0 +1,219 @@
+// @flow
+import React from 'react';
+
+import config from 'config';
+import Link from 'components/generic/Link';
+import loadable from 'higherOrder/loadable';
+import { schemaProcessDataPageSection } from 'schema_org/processors';
+
+import fonts from 'EBI-Icon-fonts/fonts.css';
+import ipro from 'styles/interpro-new.css';
+import ebiGlobalStyles from 'ebi-framework/css/ebi-global.css';
```

Comment:
Similar to the comment in TooltipAndRTDLink: I'm trying to remove the use of `ebi-global.css`, most of the styling there overwrites foundation styles, so when foundation is removed it is not necessary. But we need to check it looks ok when it's out